libs/editor/egui_editor/src/galleys.rs

```diff
@@ -121,7 +121,8 @@ pub fn calc(
             // see https://github.com/lockbook/lockbook/issues/1983, https://github.com/emilk/egui/issues/3203
```

Comment:
there's movement on this issue and the egui bug is tagged for egui's 0.23 release https://github.com/emilk/egui/issues/3203#issuecomment-1717408018